tests/dummy/app/components/tutorial-4/component.js

```diff
@@ -1,33 +1,40 @@
-import TutorialComponent from '../shared-tutorial/component';
+import TutorialComponent from "../shared-tutorial/component";
 
 // BEGIN-SNIPPET better-syntax-5
-import { action } from '@ember/object';
+import { action } from "@ember/object";
 
 export default class Tutorial4 extends TutorialComponent {
   result = null;
   isFindingStores = false;
 
   @action
   async findStores() {
-    if (this.isFindingStores) { return; }
+    if (this.isFindingStores) {
+      return;
+    }
 
     let geolocation = this.geolocation;
     let store = this.store;
 
-    this.set('isFindingStores', true);
+    this.set("isFindingStores", true);
 
-    try {                                   // ++
-      let coords = await geolocation.getCoords()
+    try {
```

Comment:
We should make sure the `// ++` lines in this try/catch block line up here too to the lines that are added in this step of the tutorial